src/handlers/base.py

```
@@ -0,0 +1,111 @@
+"""
+Base handler interface for processing Kafka messages.
+"""
+import abc
+import logging
+from typing import Any, Callable, Dict
```

Ruff (0.8.2)

`typing.Callable` imported but unused

Remove unused import: `typing.Callable`

(F401)